webapps/frontend/app/viewmodels/wizards/addvpool/gather_vpool.js

```diff
@@ -446,8 +448,10 @@ define([
                             }
                         });
                         if (maxCacheQuota !== 0) {
-                            self.data.cacheQuota(maxCacheQuota / Math.pow(1024.0, 3));
-                            self.data.cacheQuotaConfigured(true);
+                            self.data.cacheQuotaFC(maxCacheQuota / Math.pow(1024.0, 3) / 2);
```

Comment:
Should only be divided by 2 in case both fragment + block caching is used
Also the calculation for `maxCacheQuota` won't work